Gems/LandscapeCanvas/Code/Source/LandscapeCanvasSystemComponent.h

```diff
@@ -42,6 +44,11 @@ namespace LandscapeCanvas
         void NotifyRegisterViews() override;
         ////////////////////////////////////////////////////////////////////////
 
+        ////////////////////////////////////////////////////////////////////////
+        // AztoolsFramework::ActionManagerRegistrationNotificationBus::Handler overrides
```

Comment:
nit: I know this is matching the existing style but it might be nice to just simplify this (and other comments to `AzToolsFramework::ActionManagerRegistrationNotificationBus overrides ...` (and remove all the ///////////// lines)